### PR TITLE
LVRUBY-47: Some fixes in views added

### PR DESCRIPTION
**Pull request by PidtserkovnyiVitalii**
## JIRA ticket

[LVRUBY-47](http://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=979&view=detail&selectedIssue=LVRUBY-47&quickFilter=2834)

## Code reviewers

- [ ] @omaik 
- [ ] @kmandziy 

## Summary of issue

Add link to news in header
Add created_at time to news
Add created_at time to comments
Fix i18n link in footer
Add links to user profile at news index page , and news show page
Add links to user profile at comments

## Summary of change

Added link to news in header
Added created_at time to news
Added created_at time to comments
Fixed i18n link in footer
Added links to user profile at news index page , and news show page
Added links to user profile at comments

## Review checklist

- [x] Rubocop doesn't raise any warnings and offenses
- [x] New features are covered with new specs
- [x] All specs pass

### CODE REVIEW

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.13%** covered at **4.38** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.13%** covered at **4.38** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **97.13%** covered at **4.38** hits/line

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,4 @@
 /coverage
 *.rdb
 /public/uploads/*
+/config/.DS_Store
```
Comment (omaik) [inline .gitignore]: no empty line at the end of file

File: app/views/comments/_comment.html.haml [hunk 2]
```diff
@@ -1,7 +1,9 @@
 .container.my-border
   %strong
-    = comment.user.first_name
-    = comment.user.last_name
+    = link_to "#{comment.user.first_name} #{comment.user.last_name}",
+               user_path(comment.user.id),
+               class: "text-dark"
+  = comment.created_at.in_time_zone('Europe/Kiev').strftime("%Y-%m-%d %R")
```
Comment (omaik) [inline app/views/comments/_comment.html.haml]: Hardcoded timezone is a bad approach, it should be configured on server/user level

Comment (omaik) [reply app/views/comments/_comment.html.haml]: Also, this logic should not be in view

File: app/views/news/index.html.haml [hunk 3]
```diff
@@ -19,9 +19,13 @@
               = news.short_information
             %p.mb-1
               = t ('news.links.written_by')
-              %strong.text-dark#name
-                = news.user.first_name
-                = news.user.last_name
+              %strong
+                = link_to "#{news.user.first_name} #{news.user.last_name}",
```
Comment (omaik) [inline app/views/news/index.html.haml]: it is probably better to define method `full_name` in user

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.18%** covered at **3.91** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.18%** covered at **3.91** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.18%** covered at **3.91** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **94.93%** covered at **3.89** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **94.93%** covered at **3.89** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **94.93%** covered at **3.89** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **94.93%** covered at **3.89** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **94.69%** covered at **3.86** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **94.69%** covered at **3.86** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.63%** covered at **3.84** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.63%** covered at **3.84** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.63%** covered at **3.84** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.05%** covered at **3.76** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.05%** covered at **3.76** hits/line

**Pull request merged**